.pre-commit-config.yaml

```diff
@@ -33,12 +35,24 @@ repos:
         language: fail
         files: '[a-zA-Z0-9_]*/i18n/en\.po$'
   - repo: https://github.com/oca/maintainer-tools
-    rev: ab1d7f6
+    rev: 969238e47c07d0c40573acff81d170f63245d738
     hooks:
       # update the NOT INSTALLABLE ADDONS section above
       - id: oca-update-pre-commit-excluded-addons
       - id: oca-fix-manifest-website
         args: ["https://github.com/OCA/purchase-workflow"]
+      - id: oca-gen-addon-readme
+        args:
+          - --addons-dir=.
+          - --branch=14.0
+          - --org-name=OCA
+          - --repo-name=purchase-workflow
+          - --if-source-changed
+  - repo: https://github.com/OCA/odoo-pre-commit-hooks
```

Comment:
This new checker is raising several incorrect things that weren't previously detected. Can you please fix them?